### PR TITLE
Admin Account Registration Seeder-1

### PR DESCRIPTION
**Pull request by elyric-sun-asterisk-dev**
cd frontend
npm start

[Pre-Conditions]
None

[Expected Output]
Expected to display the Admin Registration page UI with route admin/register.

[Notes]
None

[Screenshots]
image

### CODE REVIEW

File: frontend/src/pages/admin/AdminRegistrationPage.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import AdminRegistrationForm from "../../components/admin/AdminRegistrationForm";
```
Comment (chevyB) [inline frontend/src/pages/admin/AdminRegistrationPage.js]: Admin dont need registration for security purposes. Create an admin account via seeder

Comment (elyric-sun-asterisk-dev) [reply frontend/src/pages/admin/AdminRegistrationPage.js]: Ahhhh okay2 sir 🙇

Comment (elyric-sun-asterisk-dev) [reply frontend/src/pages/admin/AdminRegistrationPage.js]: There will be no more Registration page and Registration Admin route. 🙇 
image
image
image

File: backend/app/Models/Admin.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * The attributes that should be cast.
+     *
+     * @var array<string, string>
+     */
+    protected $casts = [
+        'email_verified_at' => 'datetime',
+    ];
+}
```
Comment (chevyB) [inline backend/app/Models/Admin.php]: Pls add always extra one line after end of file. 

Apply to all files
You will see an icon like this
image

Comment (elyric-sun-asterisk-dev) [reply backend/app/Models/Admin.php]: copy sir 🙇

File: backend/database/seeders/DatabaseSeeder.php [hunk 3]
```diff
@@ -20,5 +22,19 @@ public function run()
...
+{
+	public function run()
+	{
+		Admin::create([
+			'name' => 'Elyric Admin',
+			'email' => 'elyric@admin.com',
+			'password' => bcrypt('password123'),
+		]);
+	}
+}
```
Comment (chevyB) [inline backend/database/seeders/DatabaseSeeder.php]: Create separate file and call that file here

File: backend/app/Models/Admin.php (hunk 2 above)
Comment (elyric-sun-asterisk-dev) [reply backend/app/Models/Admin.php]: I tried to edit sir but it automatically remove the last line 🤔

**Review by chevyB — APPROVED** [review]

**Pull request merged**